icd/api/pipeline_binary_cache.cpp

```diff
@@ -1203,7 +1202,7 @@ VkResult PipelineBinaryCache::InitLayers(
 
     bool memoryLayerOnline = (InitMemoryCacheLayer(settings) >= VK_SUCCESS);
 
-    bool archiveLayerOnline = internal && (InitArchiveLayers(pDefaultCacheFilePath, settings) >= VK_SUCCESS);
+    bool archiveLayerOnline = createArchiveLayers && (InitArchiveLayers(pDefaultCacheFilePath, settings) > VK_SUCCESS);
```

Comment:
I missed this when I reviewed this before.  Why did >= VK_SUCCESS change to > VK_SUCCESS?

Reply:
This is a typo.

Reply:
I opened a PR with a fix here: https://github.com/GPUOpen-Drivers/xgl/pull/86